src/linear_algebra/span.lean

```diff
@@ -181,6 +181,8 @@ end
 @[simp] lemma span_empty : span R (∅ : set M) = ⊥ :=
 (submodule.gi R M).gc.l_bot
 
+lemma mem_span_empty {x : M} : x ∈ span R (∅ : set M) ↔ x = 0 := by rw [span_empty, mem_bot]
```

Comment:
I don't really see the point in this lemma vs the one above; especially since `simp` can solve this.

We don't have this lemma for `closure_empty` or similar elsewhere, do we?

Reply:
Good point, I'll remove it later